run_all.py

```diff
@@ -22,56 +43,32 @@
 
 
 # text for shell script
-def bash_text(config_file):
+def bash_text(config_file, sublist):
     bash_job = """#!/bin/sh
 #SBATCH --account=def-jcohen
 #SBATCH --time=0-08:00        # time (DD-HH:MM)
 #SBATCH --nodes=1
 #SBATCH --cpus-per-task=32    # number of OpenMP processes
 #SBATCH --mem=128G
 
-sct_run_batch -config """ + str(config_file) + """
-"""
+sct_run_batch -config {} -include-list {}
```

Comment:
```suggestion
cd $SCRATCH
sct_run_batch -config {} -include-list {}
```
so it could be run from everywhere in the cluster